### PR TITLE
SCMOD-11992: Update release notes + copyright

### PR DESCRIPTION
**Pull request by xbreizh**
https://portal.digitalsafe.net/browse/SCMOD-11992

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
             <dependency>
                 <groupId>com.github.workerframework</groupId>
                 <artifactId>worker-framework</artifactId>
-                <version>4.0.0-SNAPSHOT</version>
+                <version>4.0.0</version>
```
Comment (rorytorneymf) [inline pom.xml]: This isn't right, updating this is part of https://portal.digitalsafe.net/browse/SCMOD-11991

File: release-notes-3.4.0.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
-
+- [SCMOD-8955]: Updated worker framework version
```
Comment (rorytorneymf) [inline release-notes-3.4.0.md]: Not formatted correctly, look at other examples.

Comment (rorytorneymf) [inline release-notes-3.4.0.md]: Jira number not correct

**Review by rorytorneymf — CHANGES_REQUESTED** [review]
Build is also failing

File: release-notes-3.4.0.md (hunk 2 above)
Comment (rorytorneymf) [inline release-notes-3.4.0.md]: Include the version the worker framework has been updated  to:

 Worker Framework updated to [x.x.x.](https://github.com/WorkerFramework/worker-framework/releases/tag/vx.x.x)

Comment (xbreizh) [reply release-notes-3.4.0.md]: Done

Comment (xbreizh) [reply release-notes-3.4.0.md]: sorry, you re right, done

Comment (xbreizh) [reply release-notes-3.4.0.md]: Done

File: pom.xml (hunk 1 above)
Comment (xbreizh) [reply pom.xml]: restored

File: release-notes-3.4.0.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+- [SCMOD-9988]: Update to latest worker-framework 3.4.0
```
Comment (rorytorneymf) [inline release-notes-3.4.0.md]:
```suggestion
- SCMOD-9988: Worker Framework updated to 4.0.0

**Review by rorytorneymf — CHANGES_REQUESTED** [review]

**Review by rorytorneymf — APPROVED** [review]

**Pull request merged**